src/dune_rules/context.mli

```diff
@@ -105,8 +106,11 @@ val to_dyn_concise : t -> Dyn.t
 (** Compare the context names *)
 val compare : t -> t -> Ordering.t
 
+(** Return the OCaml compiler needed for the OCaml compilation mode *)
+val ocaml_compiler : t -> Mode.t -> Action.Prog.t
```

Comment:
Do we need this function? I think we can just use `Context.compiler` everywhere.

Reply:
Removed in https://github.com/jchavarri/dune/pull/4/commits/73108d0e600572cad907a45a9f86107dd3974f46.